yml/dp02_dc2.yaml

```diff
@@ -8238,3 +8238,174 @@ tables:
     datatype: long
     mysql:datatype: BIGINT
     description:
+- name: Visit
+  '@id': '#Visit'
+  description: ''
+  columns:
+  - name: visit
+    '@id': '#Visit.visit'
+    datatype: long
+    description: ''
+  - name: physical_filter
+    '@id': '#Visit.physical_filter'
+    datatype: char
+    length: 32
```

Comment:
I would argue that MySQL-specific type definitions were added for each column of the schema, such as:
```
mysql:datatype: CHAR(32)
```
And, perhaps, in case strings are not exactly 32-character long a better option would be:
```
mysql:datatype: VARCHAR(32)
```
This would save quite a bit of space for very large tables.